document/src/main/java/com/yahoo/document/DocumentTypeManagerConfigurer.java

```diff
@@ -159,7 +159,7 @@ private DataType createReferenceType(int id, DocumentmanagerConfig.Datatype.Refe
             return type;
         }
 
-
+        @SuppressWarnings("deprecation")
```

Comment:
If you deprecate the DocumentManagerConfigurer you do not need supressions.
It should not be public either.
But that can also be done separately.